### ISSUE TITLE
Quality selector

### ISSUE THREAD

**Issue by ddurdle** [jellyfin/jellyfin-androidtv#43]
The original Emby TV apps had the ability to change the quality (resolution, bit rate). Completely missing from this Android TV app (present on the mobile Android app).

**Comment by ddurdle** [comment]
I've updated the title and suggested this should allow the ability for the user to transcode, via a quality selector.  Right now, the latest android tv app, it is direct play or bust.

**Comment by kinv** [comment]
There is also no playback correction. I have the setting to down mix to stereo however the file just direct plays with no sound. Transcoding isn't kicked off to transcode audio.

**Comment by ddurdle** [comment]
I agree this might need to be upgraded to a bug.  There are no kickoffs to the transcoding process from content watched via this app.  The mobile version doesn't have this issue.

**Comment by thornbill** [comment]
If the down mix option is not working then we should track that as a separate issue. I agree that is a bug vs adding a quality selector which would be a new feature.

**Comment by dausruddin** [comment]
Tried installing Jellyfin Mobile instead and it was unusable LOL. Hopefully this feature will be added anytime soon.
